### ISSUE TITLE
windows: error writing a new file

### ISSUE THREAD

**Issue by yoava333** [neovim/neovim#4710]
- Neovim version: 1.5-dev
- [x] Vim behaves differently? Vim version: all
- Operating system/version: Windows 10 x64
- Terminal name/version: neovim-qt
- `$TERM`:

### Actual behaviour
when writing a new file (e.g. :w outputfile) I'm getting this error:
E503 "outputfile" is not a file or writable device

### Expected behaviour
the content of the buffer be written to disk in a file named outputfile

### Steps to reproduce using `nvim -u NORC`
```
nvim -u NORC
i
some text in the buffer<Esc>
:w outputfile
# getting error here
```

**Comment by jamessan** [comment]
Are you in a directory that you have permission to create a file in?  What does `:pwd` show?

**Comment by equalsraf** [comment]
This should be more or less fixed in #810 - see https://github.com/neovim/neovim/pull/4705 for more details. @yoava333 the latest build should be working

    https://ci.appveyor.com/project/equalsraf/neovim/build/817

**Comment by yoava333** [comment]
Can confirm that this is fixed on the latest build that @equalsraf suggested
